spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaDashboardProperties.java

```diff
@@ -16,8 +16,9 @@
 
 package org.springframework.cloud.netflix.eureka.server;
 
-import lombok.Data;
-
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
@gzurowski, @ryanjbaxter - Isn't commons-lang3 a better choice for `Equals`, `HashCode` and `ToString` - so the imports will be `org.apache.commons.lang3.builder.EqualsBuilder`. I see that commons-lang3 is already transitively imported.